src/pages/minutes.js

```diff
@@ -2,26 +2,45 @@ import { graphql } from "gatsby";
 import React from "react";
 import Layout from "../components/Layout";
 import * as styles from "../styles/standard-page.module.css";
-
+import { Accordion } from "react-bootstrap";
+import '../styles/minutes-page.css';
 
 export default function Minutes({ data }) {
 	const nodes = data.allStrapiMinutes.edges;
-	console.log(nodes);
-
+	let years = Array(0);
+	nodes.map((node) => {
+		if (!years.includes(node.node.DateOfMeeting.substring(6, 10))) {
```

Comment:
if a `minutes` has not been given a `dateOfMeeting` then it will break the whole page. 
I'd suggest making the date field `required` or have a separate tab in the accordion for undated minutes
![image](https://user-images.githubusercontent.com/25283679/172141027-7404e08c-1ca9-4b89-9e80-e74a5abace81.png)

Reply:
also, it would be best practice to have them in order of most recently updated first, and have the current year/most recent upload tab open on page load. accessibility and all that

Reply:
Will be pushed in a min